packages/@lwc/shared/src/__tests__/context.spec.ts

```diff
@@ -96,8 +98,12 @@ describe('context', () => {
             expect(isTrustedContext({})).toBe(false);
         });
 
-        it('should return true for all calls when trustedContexts is not set', () => {
-            expect(isTrustedContext({})).toBe(true);
+        it('should return false for all calls when trustedContexts is not set', () => {
+            expect(isTrustedContext({})).toBe(false);
+        });
+
+        it('legacyIsTrustedContext should return true when trustedContexts is not set', () => {
+            expect(legacyIsTrustedContext({})).toBe(true);
```

Comment:
For posterity, can we add a test that shows the legacy vs new beavior?  

Ex, a test where the flag is enabled vs disabled and the expected outputs.